src/i18n/lib.js

```diff
@@ -262,7 +265,7 @@ export function configure(options) {
   loggingService = options.loggingService;
   // eslint-disable-next-line prefer-destructuring
   config = options.config;
-  messages = Array.isArray(options.messages) ? mergeMessages(options.messages) : options.messages;
+  messages = Array.isArray(options.messages) ? merge({}, ...options.messages) : options.messages;
```

Comment:
@arbrandes - the only use of the mergeMessages function was here (and in the unit tests). Note how it has now been changed to use lodash.merge directly. The exports outside of /lib and the package itself were added in this commit.